src/Compilers/VisualBasic/Portable/Compilation/SemanticModel.vb

```diff
@@ -3311,14 +3311,14 @@ _Default:
         End Function
 
         Protected NotOverridable Overrides Function AnalyzeDataFlowCore(firstStatement As SyntaxNode, lastStatement As SyntaxNode) As DataFlowAnalysis
-            Return Me.AnalyzeDataFlow(SafeCastArgument(Of StatementSyntax)(firstStatement, "firstStatement"),
-                                                SafeCastArgument(Of StatementSyntax)(lastStatement, "lastStatement"))
+            Return Me.AnalyzeDataFlow(SafeCastArgument(Of StatementSyntax)(firstStatement, NameOf(firstStatement)),
+                                                SafeCastArgument(Of StatementSyntax)(lastStatement, NameOf(firstStatement)))
```

Comment:
@AdamSpeight2008 @gafter This appears to have been accidentally changed from lastStatement to firstStatement.